### PR TITLE
Enable environment variables to configure image

### PR DESCRIPTION
**Pull request by yongtin**

### CODE REVIEW

File: scripts/config.rb [hunk 1]
```diff
@@ -5,18 +5,16 @@
 require 'erb'
 require 'ostruct'
 
-TEMPLATE = "/root/filebeat.yml.erb"
-CONFIG = "/root/config.yml"
+TEMPLATE = ENV["FILEBEAT_TEMPLATE"] || "/root/filebeat.yml.erb"
```
Comment (roiravhon) [inline scripts/config.rb:8]: Tested this change?
It does not produce an exception in case its not present?

Comment (yongtin) [reply scripts/config.rb:8]: yup. I built an image yongtin/logzio-filebeat to verify this. 

Ran the config.rb and generated the same /filebeat.yml file with the same md5 hash as logzio/logzio-filebeat. Main diff now is I can customize the path if I wanted to via environment variables.

**Comment by roiravhon** [comment]
Hey @yongtin and sorry for the delay in the response.
We are just testing it a bit and will upload, I'll let you know when it's done.

In the meanwhile, can you document your new feature in the README as well?

**Comment by roiravhon** [comment]
Pushed to dockerhub. Thanks!

**Pull request merged**